packages/icons/src/sscs-enforce.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 28 25">>
+  <path
+    d="M21.295 20.87h.5v-.003l-.5.003ZM21.19 3.682h-.5v.003l.5-.003Zm-17.39 0h.5-.5Zm11.686 15.273h.5a.5.5 0 0 0-.5-.5v.5ZM1 18.955v-.5a.5.5 0 0 0-.5.5H1ZM27 5.639v.5a.5.5 0 0 0 .5-.5H27Zm-5.8 0-.5.003.5-.003Zm-11.12 5.645a.5.5 0 0 0-.707.707l.707-.707Zm1.147 1.854-.353.354a.5.5 0 0 0 .707 0l-.354-.354Zm1.668-1.667.353.353-.353-.353Zm1.68-.974a.5.5 0 1 0-.707-.707l.707.707Zm-2.514 1.807-.354-.353.354.353Zm8.734 8.566a2.406 2.406 0 0 1-2.405 2.406v1a3.406 3.406 0 0 0 3.405-3.406h-1Zm-2.405 2.406a2.406 2.406 0 0 1-2.405-2.406h-1a3.406 3.406 0 0 0 3.405 3.406v-1Zm-14.485 0A2.406 2.406 0 0 1 1.5 20.87h-1a3.406 3.406 0 0 0 3.405 3.406v-1Zm0 1H18.39v-1H3.905v1Zm2.8-24a3.406 3.406 0 0 0-3.406 3.406h1a2.406 2.406 0 0 1 2.406-2.406v-1Zm9.28 20.594v-1.915h-1v1.915h1ZM.5 18.955v1.915h1v-1.915h-1Zm14.985-.5H3.799v1h11.686v-1Zm-11.686 0H1v1h2.8v-1ZM27.5 3.682A3.406 3.406 0 0 0 24.095.276v1A2.406 2.406 0 0 1 26.5 3.682h1ZM24.095.276a3.406 3.406 0 0 0-3.406 3.406h1a2.406 2.406 0 0 1 2.406-2.406v-1Zm-17.4 1h17.4v-1h-17.4v1ZM3.299 3.682v15.273h1V3.682h-1Zm23.201 0V5.64h1V3.682h-1Zm-5.81.003.01 1.957 1-.006-.01-1.957-1 .006Zm.01 1.957.095 15.231 1-.006-.094-15.231-1 .006ZM27 5.14h-5.8v1H27v-1Zm-11.013 6.498a4 4 0 0 1-4 4v1a5 5 0 0 0 5-5h-1Zm-4 4a4 4 0 0 1-4-4h-1a5 5 0 0 0 5 5v-1Zm-4-4a4 4 0 0 1 4-4v-1a5 5 0 0 0-5 5h1Zm4-4a4 4 0 0 1 4 4h1a5 5 0 0 0-5-5v1Zm-2.614 4.354 1.5 1.5.708-.707-1.5-1.5-.708.707Zm3.875-.167 1.327-1.327-.707-.707-1.327 1.327.707.707Zm-1.667 1.668.834-.834-.707-.707-.834.833.707.707Zm.834-.834.833-.834-.707-.707-.834.834.708.707Z" />
+</svg>
```

Comment:
We should run svgo on icons

Reply:
https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

Reply:
Already ran the files through svgo but had prettify markup enabled.